File: ProBen.Domain/Beneficio.cs
using System;
using System.Collections.Generic;

namespace ProBen.Domain
{
    public class Beneficio
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public DateTime DataCriacao { get; set; }
        public List<Categoria> Categorias { get; set; }
        public List<Setor> Setores { get; set; }
        public List<Servidor> Servidores { get; set; }
        public string ImagemUrl { get; set; }
    }
}

File: ProBen-App/src/app/_models/Beneficio.ts
import { Categoria } from "./Categoria";
import { Servidor } from "./Servidor";
import { Setor } from "./Setor";

export class Beneficio {
    constructor() {}

    id!: number;
    nome!: string;
    dataCriacao!: Date;
    categorias!: Categoria[];
    setores!: Setor[];
    servidores!: Servidor[];
    imagemUrl!: string;
}
